src/hand_demo_warehouse.cpp

```diff
@@ -1,4 +1,12 @@
 #include <ros/ros.h>
+#include <sensor_msgs/Image.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+#include <message_filters/subscriber.h>
+#include <message_filters/time_synchronizer.h>
```

Comment:
Now this code depends on OpenCV and message_filters, but does not use either.
Looks like you moved this to `save_data.cpp` later but still commited some changes here... :/